docs/source/classroom/class_guide.md

```diff
@@ -0,0 +1,172 @@
+
+(sbatch-courses-index)=
+### sbatch script
+
+An sbatch script can be submitted on the command line via the command `sbatch scriptname.sh`. Below are some examples of sbatch scripts using the courses and courses-gpu partitions. See {ref}`slurm-running-jobs` for more information on running sbatch scripts or run `man sbatch` for additional sbatch parameters.
+
+::::{dropdown} courses partition
+:::{code-block}bash
+
+#!/bin/bash
+
+#SBATCH --nodes=1  
+#SBATCH --time=4:00:00  
+#SBATCH --job-name=MyJobName  
+#SBATCH --partition=courses  
+#SBATCH --mail-type=ALL  
+#SBATCH --mail-users=$USER@northeastern.edu  
+
+#commands to execute  
+
+:::
+::::
+
+
+::::{dropdown} courses-gpu partition
+:::{code-block}bash
+
+#!/bin/bash
+
+#SBATCH --nodes=1  
+#SBATCH --time=4:00:00  
+#SBATCH --job-name=MyJobName  
+#SBATCH --partition=courses-gpu  
+#SBATCH --gres=gpu:1  
+#SBATCH --mail-type=ALL  
+#SBATCH --mail-users=$USER@northeastern.edu  
+
+#commands to execute  
+
+:::
+::::
+
+(srun-courses-index)=
+### srun interactive session
+
+An interactive session can be run on the command line via the `srun` command as shown in the examples below. See {ref}`slurm-running-jobs` for more information on using `srun` or run `man srun` to see additinal parameters that can be set with `srun`.
+
+
+::::{dropdown} courses partition
+:::{code-block} bash
+srun --time=4:00:00 --job-name=MyJob --partition=courses --pty /bin/bash
+:::
+::::
+
+::::{dropdown} courses-gpu partition
+:::{code-block} bash
+srun --time=4:00:00 --job-name=MyJob --partition=courses-gpu --gres=gpu:1 --pty /bin/bash
+:::
+::::
+
+(OOD-courses-index)=
+### Open OnDemand
+
+We have several widely-used applications available on the Open OnDemand (OOD) website including, Jupyterlab Notebook, Rstudio, Matlab, GaussView and more. {ref}`access-ood`
+
+All of the applications under the “Courses” tab on the dashboard can be set to either the `courses` or `courses-gpu` partitions via the applications specific pull down menus.
+
+### Monitoring Jobs
+
+Whichever way you choose to run your jobs, you can monitor their progress with the command `squeue`.
+
+:::{code-block} bash
+squeue -u username
+:::
+
+You can also monitor jobs being run on either of the courses partitions.
+
+:::{code-block} bash
+squeue -p courses
+squeue -p courses-gpu
+:::
+
+## Software Applications
+---
+All courses have access to the [command line](https://rc-docs.northeastern.edu/en/latest/first_steps/usingbash.html#command-line).
+
+We have many software applications installed system wide as modules that are available through the command line via the [“module” command](https://rc-docs.northeastern.edu/en/latest/software/systemwide/modules.html).
+
+We also support many software applications for courses and have interactive versions on the [Open OnDemand](https://rc-docs.northeastern.edu/en/latest/using-ood/index.html) website including:
+
+Jupyterlab notebook, Rstudio, Matlab, VSCode, Maestro (Schrodinger), and a unix Desktop
```

Comment:
I have seen unix written as unix, Unix, and UNIX (not in our RTD but just out and about). Not sure if there is a more correct one than another but we should definitely decide and stick with one in terms of the RTD and the OOD side of things.